docker/cla_backend.ini

```diff
@@ -11,11 +11,9 @@ processes = 2
 chdir = /home/app/django
 env = DJANGO_SETTINGS_MODULE=cla_backend.settings
 module = cla_backend.wsgi:application
-req-logger = file:/var/log/wsgi/request.log
-logger = file:/var/log/wsgi/error.log
 post-buffering = 1
 harakiri = 20
 buffer-size=32768
 post-buffering-bufsize=32768
-
-cron = 0  5 -1 -1 0 python /home/app/django/manage.py housekeeping
+cron = 0 5 -1 -1 0 python /home/app/django/manage.py housekeeping
+cron = 15 2 -1 -1 -1 python /home/app/django/manage.py monitor_missing_outcome_codes
```

Comment:
For posterity, could you link the original location where it was moved from, please?